packages/cli/src/CommandsLogic.ts

```diff
@@ -315,12 +315,11 @@ export class CommandsLogic {
           throw new Error(`Given minimum unstake delay ${options.unstakeDelay.toString()} too low for the given hub ${config.minimumUnstakeDelay.toString()}`)
         }
         const stakeValue = toBN(options.stake.toString()).sub(stake)
-        // TODO: do not approve PR if this line is here - test this change manually!
-        console.log(`Staking relayer ${fromWei(stakeValue, 'ether')} eth`,
-          stake.toString() === '0' ? '' : ` (already has ${fromWei(stake, 'ether')} eth)`)
+        console.log(`Staking relayer ${fromWei(stakeValue, 'ether')} ${tokenSymbol}`,
```

Comment:
you can't assume "ether" style digits. you must call `decimals()`
e.g. USDC is only 6 decimals
better have a `parseTokenValue(amount, token)` method.

Reply:
Noted and implemented. Did not know there are self-respecting tokens that override 'decimals'.